src/screens/index.tsx

```diff
@@ -1,5 +1,5 @@
 import Home from "./Home";
 import MapList from "./MapList";
-import Info from "./Info";
+import Info from "./InfoScreen/InfoScreen";
 
 export { Home, MapList, Info };
```

Comment:
For consistency, I think all of the imports/exports should be renamed as well: import InfoScreen, export { HomeScreen } etc.

Reply:
good catch thank you

Reply:
You're right, I will adjust the import for InfoScreen throughout the app. We can adjust Home to be HomeScreen when we have a ticket out to adjust Home to be HomeScreen, etc.